### PR TITLE
feat: show tier modal on account page after logging in

### PR DESCRIPTION
**Pull request by jsdevel**
This shows the user the plan they wanted from the home page after logging in.

### CODE REVIEW

File: packages/website/pages/account/payment.js
Comment (gobengo) [inline packages/website/pages/account/payment.js:5]: I think this will bundle in a browser polyfill of this thing from nodejs stdlib. wdyt of using the DOM URLSearchParams constructor? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams

Comment (jsdevel) [reply packages/website/pages/account/payment.js:5]: @gobengo i looked and it looks like we're already bundling it.  i can follow up with a tech debt PR if you want to refactor all locations in the codebase to use `URL`.
querystring

File: packages/website/pages/account/payment.js
Comment (gobengo) [inline packages/website/pages/account/payment.js:201]: Is this inteded to wipe the query param?

If so, I think it's kinda nice to keep it around. That way the user can share the URL to that tier to themselves or others (e.g. via user-agent share functionality, copy url bar and paste elsewhere, etc)

Just curious, what is the motivation of the pushState and, if I'm reading it right, clearing the query param? If it's not strictly needed we might as well not do it.

Comment (jsdevel) [reply packages/website/pages/account/payment.js:201]: so, if the user closes the modal, it's their intention _not_ to see it again.  if they refresh the browser after closing the modal and then see the modal again, i figured it would be a little annoying.  if they click the browser back button they'll see the modal and query param again.  lmk what you want.

Comment (gobengo) [reply packages/website/pages/account/payment.js:201]: ah, it's onclose. Nice. Sry for missing that.

**Review by gobengo — APPROVED** [review]
Approach looks great. Approving so as to not block, but probably best to ensure the tsc check passes before merging. re-request review when it's green if you want a pair of eyes on whatever the fix is.

File: packages/website/pages/account/payment.js
Comment (gobengo) [reply packages/website/pages/account/payment.js:201]: >  if they click the browser back button they'll see the modal and query param again with my current implementation.

awesome

File: packages/website/components/accountPlansModal/accountPlansModal.js
Comment (gobengo) [inline packages/website/components/accountPlansModal/accountPlansModal.js:37]: nice, I didn't realize all these `any` were in there :P

File: packages/website/pages/account/payment.js
Comment (gobengo) [inline packages/website/pages/account/payment.js:49]: love how the function name is self-documenting 🙏

**Review by gobengo — APPROVED** [review]

**Comment by jsdevel** [comment]
> Approach looks great. Approving so as to not block, but probably best to ensure the tsc check passes before merging. re-request review when it's green if you want a pair of eyes on whatever the fix is.

yea, pushed up some changes to make the build pass.  fingers crossed...

**Pull request merged**
